components/automate-chef-io/content/docs/datafeed.md

```diff
@@ -0,0 +1,138 @@
++++
+title = "Data Feeds"
+description = "Export Node Data from Chef Client Runs to External Data Collectors"
+date = 2020-05-05T13:19:02-07:00
+draft = false
+bref = ""
+toc = true
+[menu]
+  [menu.docs]
+    parent = "settings"
+    weight = 15
++++
+
+The Data Feed service sends node data to a 3rd party service.
+This can be useful when updating Configuration Management Databases, external Security Dashboards and IT service management platforms.
+The following types of information are sent:
+
+- Ohai data gathered from each managed node - This data includes hardware, operating system, and installed program information. Some variation depends on the managed operating system
```

Comment:
Throughout the document there is an odd mixture of sentences that end with a period and those that do not. (Seems like there is contention between sentences-should-have-periods and bulleted-list-items-should-not, but that makes it rather confusing.) My suggestion would be that all sentences should end with periods.

Reply:
Suggestion noted, but the format stands. A multi-sentence bullet list item will have periods to designate the end of each sentence. The last sentence in a single sentence bulleted list item or multi-sentence bulleted list item does not end with a period.